### DISCUSSION TITLE
performance-reviews

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#12] [Giscus]
# performance-reviews

Feedback is critical for alignment and growth, but really awkward. How should small, early stage companies handle performance reviews? This article gives some best practices from my perspective doi...

https://matthewrocklin.com/performance-reviews.html

**Comment by JulianWgs** [comment]
Hi Matt,

great insights into performance reviews :) I really like the idea of of gathering feedback from colleagues, scrambling it and then using this as the basis for my own feedback. Did the peer review process at Valve inspire you to do this?

I would like to comment on the infamous feedback burger. This concept was taught to me multiple times in university and is broadly accepted. However a few years ago more and more people (mostly psychology students) have recommended me the "Wahrnehmung (perception/observation), Wirkung (effect/interpretation), Wunsch (request/wish)" framework or WWW framework. 

The problem with the feedback burger is that is confuses the recipient. First they get excessive praise (even if there is little to build upon). Then it tilts to criticism, but the recipient is not in the mood, because they just heard how good they are doing. After hearing only criticism (most often than not the majority of the talk or the reason for the feedback) they is exhausted. Then the positive feedback at the end, leads them to think it wasn't so bad after all.

The WWW framework is different as it focuses on what you trying to achieve: Change in behavior (or on how you perceive it).

**Wahrnehmung (literally perception, but more like observation)**:
What did you actually observe? What did you hear, saw or read? This step is about facts. Don't interpret anything into these observation just yet.

Examples:
- "Today, you didn't send the email in time"
- "In the presentation on Friday you showed each slide only for brief time"

**Wirkung (effect/interpretation)**:
How did these observations affect you? How do you interpret the given facts? Here you can't really do anything wrong beside being unkind. Just don't forget to formulate everything from your perspective.

Examples:
- "It seams to me that writing code is more important to you than sending emails"
- "I couldnt follow the presentation".

**Wunsch (literally wish, but more like request)**:
What behavior do you want from the recipient? Formulate this request not as an order, but more like kind request or wish. Its about embracing that the feedback or some assertions in it could be wrong. May be they doesn't agree with interpretation of the observation and therefore don't want to change their behavior. That`s usually fine. They know what you observed and how you interpreted that and stop letting you observe this behavior or giving additional context so you don't misinterpret it.

Examples:
- "I would like to see you send emails in time in the future."
- "I would have liked you to explain relevant keywords of the slide and thus give me time to understand"

It takes some practice to get right and to separate what you have observed and how it affected you. After using it for a few years I'm very happy with it and like it much more than the feedback burger. Especially when you just want to give one feedback point.

Of course if you want to stick to the feedback burger, you can combine these two methods as well. Unfortunately I couldn't find any additional material in English, however there is plenty of material in German and deepl should translate it quite well.

Let me know what you think :)

Greetings
Julian
